src/constraints/outage_constraints.jl

```diff
@@ -76,7 +83,8 @@ function add_outage_cost_constraints(m,p)
         @constraint(m,
             m[:dvMGStorageUpgradeCost] >= p.s.financial.microgrid_upgrade_cost_fraction * m[:TotalStorageCapCosts] - (
                 p.s.financial.microgrid_upgrade_cost_fraction * p.third_party_factor * (
-                    sum( p.s.storage.attr[b].net_present_cost_per_kw * p.s.storage.attr[b].max_kw for b in p.s.storage.types.elec) + 
+                    sum( p.s.storage.attr[b].net_present_cost_per_kw * p.s.storage.attr[b].max_kw for b in p.s.storage.types.ac_coupled) + 
+                    sum( p.s.storage.attr[b].net_present_cost_per_kw * (p.s.storage.attr[b].max_kw + sum(p.max_sizes[t] * maximum(p.production_factor[t,:]) * p.levelization_factor[t] for t in p.techs.dc_coupled_with_storage)) for b in p.s.storage.types.dc_coupled) + 
```

Comment:
for a dc-coupled run, wouldn't `storage.attr[b].max_kw` already capture the total output of the inverter? I'm confused about why `+ sum(p.max_sizes[t] * maximum(p.production_factor[t,:]) * p.levelization_factor[t]` is needed here

Reply:
storage.attr[b].max_kw is just the kw of the battery itself, the inverter kw is a separate DV that could be larger